arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get EventHub Consumer Group for an IotHub",
+				"description": "Get EventHub Consumer Group for an IotHub",
+				"operationId": "IotHubResource_GetEventHubConsumerGroup",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"name": "resourceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "eventHubEndpointName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "groupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/EventHubConsumerGroupInfo"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			},
+			"put": {
+				"tags": ["PUT"],
+				"summary": "Add an EventHub Consumer Group to an IotHub",
+				"description": "Add an EventHub Consumer Group to an IotHub",
+				"operationId": "IotHubResource_AddEventHubConsumerGroup",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"name": "resourceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "eventHubEndpointName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "groupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/EventHubConsumerGroupInfo"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/jobs": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get all the jobs in an IotHub",
+				"description": "Get all the jobs in an IotHub",
+				"operationId": "IotHubResource_GetJobs",
```

Comment:
IotHubResource_ListJobs; apply the paging extension and fix the model class name